core/src/main/java/bisq/core/offer/OfferUtil.java

```diff
@@ -364,6 +364,53 @@ public void validateOfferData(double buyerSecurityDeposit,
                 Res.get("offerbook.warning.paymentMethodBanned"));
     }
 
+    // Returns an edited payload: a merge of the original offerPayload and
+    // editedOfferPayload fields.  Mutable fields are sourced from
+    // mutableOfferPayloadFields param, e.g., payment account details, price, etc.
+    // Immutable fields are sourced from the original openOffer param.
```

Comment:
JavaDoc or it doesn't show up in the documentation popup in your IDE
```suggestion
    /**
     * Returns an edited payload: a merge of the original offerPayload and
     * editedOfferPayload fields. Mutable fields are sourced from
     * mutableOfferPayloadFields param, e.g., payment account details, price, etc.
     * Immutable fields are sourced from the original openOffer param.
     */
```